### ISSUE TITLE
No feedback when creating an account

### ISSUE THREAD

**Issue by ludoboludo** [Shopify/dawn#1065]
**Describe the current behavior**

>when a customer creates an account (using the `create_customer' form). It seems like the default behaviour is to redirect to the homepage. There is not confirmation spoken aloud by the screen reader

**Describe the expected behavior**

When going through the form, there should be some feedback so the user can know what happened.

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 2.5.0

**Possible solution**

When creating the account there is a redirect happening to the home page. We will need to somehow post there a status update confirming the creation was successful. Not sure if there is some platform changes needed or if it can be tackled in the theme only. 

**Additional context/screenshots**
[video](https://screenshot.click/21-23-v9pe4-qouip.gif)

cc: @svinkle

**Comment by Stephanie-Chou** [comment]
We are closing this issue. This issue was reported 2 years ago on an old version of Dawn. If this is still a high priority issue today, please re open with updated reproduction steps
